src/components/image/notes.stories.mdx

```diff
@@ -0,0 +1,66 @@
+
+import Image from "./"
+import { Meta, Preview, Props, Story } from "@storybook/addon-docs/blocks"
+
+<Meta title="Components/Image" component={Image} />
+
+# Image
+
+The image tag is a utility wrapper that determines how to properly render an image. It also provides the ability to set a background texture (example below).
+
+## Usage
+
+The `src` attribute is what tells the image component how to render the image. See below for examples.
+
+### Gatsby Image
+
+If the `src` is a properly-formatted object, it will be rendered using [gatsby-image](https://www.gatsbyjs.org/packages/gatsby-image/). This occurs if the object contains a key called `childImageSharp`, which itself contains either a `fluid` or `fixed` object.
+
+<Preview>
+  <Story name="Image">
```

Comment:
I would call this "Gatsby Image" so it's clear in the sidebar what the difference between the two are.